applications/openshift/worker/file_groupowner_worker_ca/rule.yml

```diff
@@ -13,8 +13,8 @@ rationale: |-
 
 severity: medium
 
-#identifiers:
-#    cce@ocp4: 80619-0
```

Comment:
@redhatrises was this CCE added as a place holder?
Is the CCE valid and assigned tot this rule actually?

Reply:
no, it's not a valid CCE. It is a placeholder since the content is still considered alpha / beta/